embed-code-go/fragmentation/fragment.go

```diff
@@ -133,7 +133,7 @@ func (fragment Fragment) text(lines []string, configuration configuration.Config
 		}
 		cutIndentLines := indent.CutIndent(line, commonIndentation)
 		for _, cutIndentLine := range cutIndentLines {
-			text += cutIndentLine
+			text += cutIndentLine + "\n"
```

Comment:
Above you use other technic do we really need the last `\n` here? and if we need it why cant we say :
```
text+ := strings.Join(cutIndentLines, "\n") + "\n"
```
To keep it the same style as above.